libr/include/r_util/r_annotated_code.h

```diff
@@ -79,105 +79,92 @@ typedef struct r_annotated_code_t {
- * @annotation: Pointer to an annotation.
- * 
  * This function recognizes the type of the specified annotation and returns true if its
  * type is any of the following three: R_CODE_ANNOTATION_TYPE_GLOBAL_VARIABLE,
  * R_CODE_ANNOTATION_TYPE_CONSTANT_VARIABLE, R_CODE_ANNOTATION_TYPE_FUNCTION_NAME
+ * @brief Checks if the specified annotation is a reference.
  * 
- * Return: Returns true if the specified annotation is a reference.
+ * @param annotation Pointer to an annotation.
+ * @return Returns true if the specified annotation is a reference.
  */
 R_API bool r_annotation_is_reference(RCodeAnnotation *annotation);
 /**
- * r_annotation_is_variable() - Checks if the specified annotation is a function variable.
- * @annotation: Pointer to an annotation.
- * 
  * This function recognizes the type of the specified annotation and returns true if its
  * type is any of the following two: R_CODE_ANNOTATION_TYPE_LOCAL_VARIABLE,
  * R_CODE_ANNOTATION_TYPE_FUNCTION_PARAMETER
+ * @brief Checks if the specified annotation is a function variable.
  * 
- * Return: Returns true if the specified annotation is a function variable.
+ * @param annotation Pointer to an annotation.
+ * @return Returns true if the specified annotation is a function variable.
  */
 R_API bool r_annotation_is_variable(RCodeAnnotation *annotation);
 /**
- * r_annotated_code_add_annotation() - Inserts *annotation in *code.
- * @code: Pointer to a RAnnotatedCode.
- * @annotation: Pointer to a annotation.
- *
  * This functions inserts the annotation represented by the pointer 'annotation' to the vector
  * of annotations in the RAnnotatedCode represented by 'code'. To be more precise,
  * annotation will be added to code->annotations, which is a RVector<RCodeAnnotation> annotations.
+ * @brief Inserts *annotation in *code.
  * 
- * Return: Nothing.
+ * @param code Pointer to a RAnnotatedCode.
+ * @param annotation Pointer to an annotation.
  */
 R_API void r_annotated_code_add_annotation(RAnnotatedCode *code, RCodeAnnotation *annotation);
 /**
- * r_annotated_code_annotations_in() - Returns all annotations with range that contains the given offset.
- * @code: Pointer to a RAnnotatedCode.
- * @offset: Offset.
- *
  * Creates an RPVector and inserts the pointers to all annotations in which 
  * annotation->start <= offset < annotation->end.
+ * @brief Returns all annotations with range that contains the given offset.
  * 
- * Return: Pointer to the RPVecrtor created.
+ * @param code Pointer to a RAnnotatedCode.
+ * @param offset Offset.
+ * @return Pointer to the RPVector created.
  */
 R_API RPVector *r_annotated_code_annotations_in(RAnnotatedCode *code, size_t offset);
 /**
- * r_annotated_code_annotations_range() - Returns all annotations with range that overlap with the given range.
- * @code: Pointer to a RAnnotatedCode.
- * @start: Start of the range(inclusive).
- * @end: End of the range(exclusive).
- *
  * Creates an RPVector and inserts the pointers to all annotations whose 
```

Comment:
In comments don't describe how, describe what the code does. "Creates an RPVector and inserts pointers" is more of literal description of code written. The brief part should be sufficient. in this case. Range endpoint behavior is already described in parameter description.